[11-May-2025 04:44:13 UTC] PHP Fatal error:  Uncaught TypeError: Illuminate\Foundation\Exceptions\Handler::renderHttpException(): Argument #1 ($e) must be of type Symfony\Component\HttpKernel\Exception\HttpExceptionInterface, TypeError given, called in E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php on line 816 and defined in E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php:889
Stack trace:
#0 E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(816): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(TypeError))
#1 E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(706): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(TypeError))
#2 E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(594): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(TypeError))
#3 E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(219): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(TypeError))
#4 E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(196): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(TypeError))
#5 E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(256): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(TypeError))
#6 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(Object(TypeError))
#7 {main}
  thrown in E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php on line 889
[11-May-2025 04:44:14 UTC] PHP Fatal error:  Uncaught TypeError: Illuminate\Foundation\Exceptions\Handler::renderHttpException(): Argument #1 ($e) must be of type Symfony\Component\HttpKernel\Exception\HttpExceptionInterface, Symfony\Component\ErrorHandler\Error\FatalError given, called in E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php on line 816 and defined in E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php:889
Stack trace:
#0 E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(816): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\ErrorHandler\Error\FatalError))
#1 E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(706): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\ErrorHandler\Error\FatalError))
#2 E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(594): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\ErrorHandler\Error\FatalError))
#3 E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(219): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\ErrorHandler\Error\FatalError))
#4 E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(196): Illuminate\Foundation\Bootstrap\HandleExceptions->renderHttpResponse(Object(Symfony\Component\ErrorHandler\Error\FatalError))
#5 E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(232): Illuminate\Foundation\Bootstrap\HandleExceptions->handleException(Object(Symfony\Component\ErrorHandler\Error\FatalError))
#6 E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(256): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#7 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}()
#8 {main}
  thrown in E:\local_projects\Laravel_projects\jobizzApp\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php on line 889
